test/e2e/kappcontroller/package_repo_test.go

```diff
@@ -176,9 +176,14 @@ spec:
 	})
 
 	logger.Section("check PackageMetadata/Packages created", func() {
-		kubectl.Run([]string{"get", "pkgm/pkg.test.carvel.dev"})
-		kubectl.Run([]string{"get", "pkg/pkg.test.carvel.dev.1.0.0"})
-		kubectl.Run([]string{"get", "pkg/pkg.test.carvel.dev.2.0.0"})
+		verify := func(resourceName string) {
+			kctlOutput := kubectl.Run([]string{"get", resourceName, "-o", "yaml"})
+			assert.True(t, strings.Contains(kctlOutput, "kapp.k14s.io/identity:"), fmt.Sprintf("unable to find identity annotation in %s", resourceName))
```

Comment:
im pretty sure assert package has some string contains function that will produce error messages automatically (instead of us having to write fmt.Sprintf...).